src/main/scala/org/allenai/scholar/metrics/metadata/BibliographyErrorAnalysis.scala

```diff
@@ -14,17 +16,25 @@ object BibliographyErrorAnalysis {
 
   val BIBLIOGRAPHY_EXTRACTION = "bibliography"
 
-  def extractBibliography(bibs: Set[PaperMetadata]): Iterable[PaperMetadata] =
+  def extractBibliography(bibs: Set[PaperMetadata]): Iterable[FuzzyPaperMetadata] =
     bibs.map(fuzzify)
 
   // Soft version of paper metadata:
-  // includes just the first word of the title and the last names of the first two authors
   def fuzzify(data: PaperMetadata) =
-    PaperMetadata(
-      Title(data.title.normalized.text.splitOnWhitespace.head),
-      Venue(""),
+    FuzzyPaperMetadata(
+      Title(data.title.normalized.text),
       data.year,
-      data.authors.map(_.lastNameOnly.normalized).take(2)
+      data.authors.map(_.lastNameOnly.normalized).toSet
     )
 
+  case class FuzzyPaperMetadata(title: Title, year: Year, authors: Set[Author]) {
```

Comment:
Extracted bibliography entries are considered a match if either the normalized title matches or the authors and year matches.